Update manage-deployments-as-code.md (#2609)
astronomer/docs: astro/manage-deployments-as-code.md

You can create, update, or delete environment variables in the `environment_variables` section of the template file. This is equivalent to configuring environment variables in the **Variables** page of a Deployment in the Cloud UI.

> When you inspect a Deployment, the value of any environment variable that is set as secret in the Cloud UI will not appear in the template file. To set any new or existing environment variables as secret in the file, specify `is_secret: true` next to the key and value. If you commit your deployment config file to source control, ensure you are not committing sensitive values in plain-text. You can either remove those secrets manually before you commit, or encrypt the file and commit the encrypted version to source control using a system like `gpg`

Reviewer: ```suggestion
When you inspect a Deployment, the value of any environment variable set as secret in the Cloud UI will not appear in the template file. To set any new or existing environment variables as secret in the file, specify `is_secret: true` next to the key and value. If you commit your deployment config file to source control, make sure you don't commit sensitive values in plain text. You can either remove those secrets manually before you commit or encrypt the file and commit the encrypted version to source control using a tool such as `gpg`.
```
